tests/conftest.py

```diff
@@ -96,19 +96,24 @@ def cleanup_test_databases(config: ConfigFile, postgres_db_engine: sa.Engine) ->
 def dbos(
     config: ConfigFile, cleanup_test_databases: None
 ) -> Generator[DBOS, Any, None]:
+    DBOS.clear_global_instance()
```

Comment:
Why do we need to clear twice?

Reply:
Just in case it was not cleared.  It keeps a test failure from cascading to the next test depending on what went wrong.  If no prior test left things in a funky state, it's a no-op.